build fail if i turn OFF pulsar.

<!-- You have to provide the following information whenever possible. -->

**Describe what's wrong**
<!-- A clear and concise description of what works not as it is supposed to. -->
```
proton/proton/src/Storages/ExternalStream/parseShards.h:6:1: error: use of undeclared identifier 'std'
    6 | std::vector<int32_t> parseShards(const std::string & shards_setting);
      | ^
proton/proton/src/Storages/ExternalStream/parseShards.h:6:13: error: use of undeclared identifier 'int32_t'
    6 | std::vector<int32_t> parseShards(const std::string & shards_setting);
      |             ^
proton/proton/src/Storages/ExternalStream/parseShards.h:6:40: error: use of undeclared identifier 'std'
    6 | std::vector<int32_t> parseShards(const std::string & shards_setting);
      |                                        ^
3 errors generated.
```

https://github.com/timeplus-io/proton/actions/runs/12602608270/job/35128518730#step:5:7041

--- a/src/Storages/ExternalStream/parseShards.h
+++ b/src/Storages/ExternalStream/parseShards.h
@@ -1,5 +1,8 @@
 #pragma once
 
+#include <vector>
+#include <string>
+
 namespace DB
 {
 

